versioned_docs/version-2.6/reference-guides/monitoring-v2-configuration/receivers.md

```diff
@@ -237,6 +237,18 @@ It is also possible to send alerts to multiple notification systems. One way is
 
 You can also set up multiple receivers by using the `continue` option for a route, so that the alerts sent to a receiver continue being evaluated in the next level of the routing tree, which could contain another receiver.
 
+## Configuring AlertmanagerConfig After A Chart Upgrade
+
+If you use SMS or Microsoft Teams alerts, and upgrade the Monitoring chart from 100.1.3+up19.0.3 to 100.2.0+up40.1.2, you may no longer receive alerts. This is due to a change that helps prevent naming collisions.
+
+You must manually edit the configuration map to continue to receive alerts:
+
+1. Click on **☰** and select **local**.
+1. Select **More Resources** > **Core** > **ConfigMaps**.
+1. Find the row corresponding to the ConfigMap for **Sachet** (for SMS alerts) or **prom2teams** (for Microsoft Teams alerts). Click the **⋮** and select **Edit YAML**.
+1. Update the delimiters in the receiver's `name` field, replacing */* characters with *-*. For example, if the receiver's name is `Test-Alert`, update it to `Test/Alert`.
```

Comment:
This should be: replacing *-* characters with */*

This is still slightly confusing as you can have receivers with '-' following the \<namespace\>/\<alertManagerConfig\>/\<receiverName\> format: default/amc-7/prom2sms. 


Can we instead say: 
Update receiver's `name` field to the following format: `\<namespace\>/\<alertManagerConfig\>/\<receiverName\>`, replacing *-* delimiters with */*. For example, if the receiver's name is `default-amc-test-prom2sms`, update it to `default/amc-test/prom2sms`.